### PR TITLE
update `pypi_helper.py`

### PR DESCRIPTION
**Pull request by mauwii**
- dont rename requests
- remove dash in verison (`2.3.0-rc3` becomes `2.3.0rc3`)
- read package_name instead of hardcode it

### CODE REVIEW

File: scripts/pypi_helper.py [hunk 1]
```diff
@@ -1,26 +1,26 @@
-import requests as request
+import requests
 
-import ldm.invoke._version as version
```
Comment (lstein) [inline scripts/pypi_helper.py:3]: This is a simpler version:
```
from ldm.invoke import __app_name__, __version__
```

**Review by lstein — APPROVED** [review]
LGTM. Left a suggestion on the version import.

File: scripts/pypi_helper.py [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+from ldm.invoke.__init__ import __app_name__, __version__
```
Comment (lstein) [inline scripts/pypi_helper.py:3]: This is a simpler version:
```
from ldm.invoke import __app_name__, __version__
```

Comment (mauwii) [reply scripts/pypi_helper.py:3]: kk, I was scared this would already import more 🙈

**Pull request merged**